Tasks/UniversalPackagesV0/universalpublish.ts

```diff
@@ -141,7 +111,45 @@ export async function run(artifactToolPath: string): Promise<void> {
             packageVersion: version,
         } as artifactToolRunner.IArtifactToolOptions;
 
-        publishPackageUsingArtifactTool(publishDir, publishOptions, toolRunnerOptions);
+        publishStatus = publishPackageUsingArtifactTool(publishDir, publishOptions, toolRunnerOptions);
+
+        //If package already exist, retry with a newer version
+        if (publishStatus != null && publishStatus == 17) {
```

Comment:
Why 17?

Reply:
Maybe it is worthwhile to use a constant to make this easier to read:
packageAlreadyExistsError = 17

Reply:
Do i define the constant on top of this file?

Reply:
not sure what is the best practice for typescript files but I'd put it on the top

Reply:
Answer for why 17: https://dev.azure.com/mseng/AzureDevOps/_git/ArtifactTool?path=%2Fsrc%2FArtifactTool%2FErrorCode.cs&version=GBmaster&_a=contents

Will add this link into PR description